util.py

```diff
@@ -1,23 +1,17 @@
-from tqdm import tqdm
-import random 
-import os
-import numpy as np
-import subprocess
-import shlex
-import sys
-
-import torch
-
-from pytorch_pretrained_bert import BertTokenizer, BertModel, BertForMaskedLM, BertForSequenceClassification, BertForNextSentencePrediction
+from pytorch_pretrained_bert import BertTokenizer, BertForSequenceClassification, BertForNextSentencePrediction
 from pytorch_pretrained_bert.optimization import BertAdam
 
 
-def load_pretrained_model_tokenizer(model_type="BertForSequenceClassification", device="cuda", chinese=False):
+def load_pretrained_model_tokenizer(model_type="BertForSequenceClassification",
+                                    local_model=None, local_tokenizer=None,
+                                    device="cuda", chinese=False):
     # Load pre-trained model (weights)
-    if chinese:
-        base_model = "bert-base-chinese"
-    else:
-        base_model = "bert-base-uncased"
+    if local_model is None:
```

Comment:
what happened when local_model is not None?

Reply:
Good catch - `local_model` was supposed to be called `base_model`. I fixed it in the latest commit.